pkgs/development/python-modules/pandas/default.nix

```diff
@@ -54,7 +53,7 @@ buildPythonPackage rec {
     xlwt
   ];
 
-  checkInputs = [ pytestCheckHook glibcLocales moto hypothesis ];
+  checkInputs = [ pytestCheckHook glibcLocales hypothesis ];
```

Comment:
In #107838, I fixed moto for python39 (because I wanted to fix pandas). Is it actually not necessary for pandas at all?

Reply:
I assumed moto requires boto which does not work on 3.9 so I went this way. Tests compile fine but I can run a review to make sure no regressions are introduced. Looking at your PR nevertheless.